content/blog/2025-10-21-datafusion-comet-0.11.0.md

```diff
@@ -0,0 +1,172 @@
+
+[change log]: https://github.com/apache/datafusion-comet/blob/main/dev/changelog/0.11.0.md
+
+## Release Highlights
+
+### Parquet Modular Encryption Support
+
+Spark supports Parquet Modular Encryption to independently encrypt column values and metadata. Furthermore, Spark supports custom encryption factories for users to provide their own key-management service (KMS) implementations. Thanks to [a](https://github.com/apache/arrow-rs/issues/7278) [number](https://github.com/apache/datafusion/issues/15216) [of](https://github.com/apache/datafusion/pull/16351) [contributions](https://github.com/apache/datafusion/pull/16779) in upstream DataFusion and arrow-rs, Comet now [supports Parquet Modular Encryption with Spark KMS](https://github.com/apache/datafusion-comet/pull/2447) for native readers, enabling secure reading of encrypted Parquet files in production environments.
+
+### Improved Memory Management
+
+Comet 0.11.0 introduces significant improvements to memory management, making it easier to deploy and more resilient to out-of-memory conditions:
+
+- **Changed default memory pool**: The default off-heap memory pool has been [changed from `greedy_unified` to `fair_unified`](https://github.com/apache/datafusion-comet/pull/2526), providing better memory fairness across operations
+- **Off-heap deployment recommended**: To simplify configuration and improve performance, Comet now expects to be deployed with Spark's off-heap memory configuration. [On-heap memory is still available](https://github.com/apache/datafusion-comet/pull/2554) for development and debugging, but is not recommended for deployment
+- **Better disk management**: The [DiskManager `max_temp_directory_size` is now configurable](https://github.com/apache/datafusion-comet/pull/2479) for better control over temporary disk usage
+- **Enhanced safety**: Memory pool operations now [use checked arithmetic operations](https://github.com/apache/datafusion-comet/pull/2455) to prevent overflow issues
+
+These changes make Comet significantly easier to configure and deploy in production environments.
+
+### Improved Apache Spark 4.0 Support
+
+Comet has improved its support for Apache Spark 4.0.1 with several important enhancements:
+
+- [Updated support from Spark 4.0.0 to Spark 4.0.1](https://github.com/apache/datafusion-comet/pull/2414)
+- [Spark 4.0 is now included in the release build script](https://github.com/apache/datafusion-comet/pull/2514)
+- Expanded ANSI mode compatibility with several new implementations:
+  - [ANSI evaluation mode arithmetic operations](https://github.com/apache/datafusion-comet/pull/2136)
+  - [ANSI mode integral divide](https://github.com/apache/datafusion-comet/pull/2421)
+  - [ANSI mode rounding functions](https://github.com/apache/datafusion-comet/pull/2542)
+  - [ANSI mode remainder function](https://github.com/apache/datafusion-comet/pull/2556)
+
+TODO: Confirm if Spark 4.0 jars are available on Maven Central.
```

Comment:
They are. You can following the link from the installation guide.

https://datafusion.apache.org/comet/user-guide/0.11/installation.html#using-a-published-jar-file